plugins/poll/test/javascripts/component/poll-results-ranked-choice-test.js

```diff
@@ -58,4 +58,16 @@ module("Poll | Component | poll-results-ranked-choice", function (hooks) {
       "displays the winner information"
     );
   });
+
+  test("Renders the ranked choice results component without error when outcome data is empty", async function (assert) {
+    this.setProperties({
+      rankedChoiceOutcome: {},
+    });
+
+    await render(
+      hbs`<PollResultsRankedChoice @rankedChoiceOutcome={{this.rankedChoiceOutcome}} />`
+    );
+
+    assert.ok(true, "No exception was thrown");
+  });
```

Comment:
I think this test should just check the texts when no outcome instead of checking for no error which seems odd.

What is the expected rendering state when no outcome?

Reply:
Done https://github.com/discourse/discourse/pull/28542/commits/acbfc61897d388ddd8e6a68f0ff80e91f603f6d3